src/acurast/app-versions.ts

```diff
@@ -0,0 +1,61 @@
+type deviceTypeNumber = number
+type buildNumber = number
+
+const androidVersions = new Map<buildNumber, string>([
```

Comment:
maybe we could either
- print this list by an info command or flag on commander
- print this list if a invalid SemVer is provided